### PR TITLE
responsive: shiftingDetails page and Breadcrumbs added phone and desktop

### LINKED ISSUE #8831: phone size responsive issue shifting details page
**Describe the bug**
responsive issue on shifting details page, 

**To Reproduce**

1. Go to 'shifting on sidebar'
2. Click on 'all details button from any card and head to shift details page'
                  'in phone size responsive issue'

**Expected behavior**
clear and better mobile user experience

**Screenshots**
Image

**Desktop (please complete the following information):**

- OS: [windows]
- Browser [chrome]
- Version [129.0.6668.101]

**Additional context**

### PR DESCRIPTION
**Pull request by modamaan**
## Proposed Changes
change the button alignment when phone size

- Fixes #8831 

Before
Screenshot 2024-10-20 000900

After
Screenshot 2024-10-20 011739

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by modamaan** [comment]
@bodhish @rithviknishad 

https://github.com/user-attachments/assets/230362f1-51a7-43dc-8260-f074e01069a3

**Comment by shivankacker** [comment]
@modamaan unable to understand the breadcrumbs relation here, could you elaborate more in the PR description?

**Comment by modamaan** [comment]
@shivankacker https://github.com/ohcnetwork/care_fe/pull/8833#issuecomment-2425499268

**Comment by modamaan** [comment]
## Proposed Changes
 add responsive shadcn breadcrumbs

- Fixes [#8831 ](https://github.com/ohcnetwork/care_fe/pull/8833#issuecomment-2425499268)

Current version
Screenshot 2024-10-27 055208

Last item broken

Screenshot 2024-10-27 055401

No dropdown here

Updated version

Screenshot 2024-10-27 055745
Last item broken issue solved

Screenshot 2024-10-27 055906
Shadcn Dropdown added

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

File: src/components/Common/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
 import { usePath, Link } from "raviger";
-import { useState } from "react";
-import { classNames } from "../../Utils/utils";
 import { Button } from "@/components/ui/button";
 import CareIcon from "../../CAREUI/icons/CareIcon";
-import useAppHistory from "@/common/hooks/useAppHistory";
+import useAppHistory from "../../common/hooks/useAppHistory";
```
Comment (bodhish) [inline src/components/Common/Breadcrumbs.tsx]: Why revert the change :( 
Lets use `@`

**Comment by bodhish** [comment]
Duplicate of https://github.com/ohcnetwork/care_fe/pull/8833

**Review by bodhish — CHANGES_REQUESTED** [review]
image
 I will actually add icons for non paths in the dropdown. 
 
 Example 
- 🏢 Critical Care
- 🗳 Assets 
> unicodes are just for illustration. 

I expect these changes to be made on the previous PR

**Comment by modamaan** [comment]
@bodhish  Screenshot 2024-10-28 193508

File: src/components/Common/Breadcrumbs.tsx (hunk 1 above)
Comment (modamaan) [reply src/components/Common/Breadcrumbs.tsx]: resolved

**Comment by modamaan** [comment]
@bodhish  i made a mistake, created branch from local branch instead of develop branch. this caused it to include all commits from my other branch, i would like to create new branch and resolve this issue

**Pull request closed without merging**